pypesto/petab/importer.py

```diff
@@ -78,6 +80,38 @@ def from_yaml(yaml_config: Union[dict, str],
             output_folder=output_folder,
             model_name=model_name)
 
+    def check_and_log_gradients(self, RTOL: float = 1e-2, ATOL: float = 1e-3):
+        """Check and log if gradients are compatible with gradient-based optimizers
```

Comment:
What is the `log` functionality of this method?

Reply:
You are right, the logging is implictly done if log level is specified to `check_grad_multi_eps`.